packages/vm/Cargo.toml

```diff
@@ -49,8 +49,8 @@ serde = { version = "1.0.103", default-features = false, features = ["derive", "
 serde_json = "1.0.40"
 sha2 = "0.10.3"
 thiserror = "1.0.26"
-wasmer = { version = "=3.2.1", default-features = false, features = ["cranelift", "singlepass"] }
-wasmer-middlewares = "=3.2.1"
+wasmer = { version = "3.3.0", default-features = false, features = ["cranelift", "singlepass"] }
+wasmer-middlewares = "3.3.0"
```

Comment:
Let's keep the `=` for wasmer. We had some trouble in the past where minor versions caused compiled module serialization to break. It's better to do those upgrades manually.

Reply:
OK. Do we move to this 3.3.0 still?